opt/scopeserver/settings.py

```diff
@@ -0,0 +1,14 @@
+# settings.py
```

Comment:
Please stick to `scopeserver/config.py`.
I would like to use [`pydantic`s `BaseSettings`](https://pydantic-docs.helpmanual.io/usage/settings/) it reads ENV vars and provides validator functions so this can be integrated into the existing `config` implementation.